mzLib/MassSpectrometry/IsobaricQuantSampleInfo.cs

```diff
@@ -0,0 +1,147 @@
+        public int BiologicalReplicate { get; }
+
+        /// <summary>
+        /// Technical replicate identifier for repeated measurements.
+        /// </summary>
+        public int TechnicalReplicate { get; }
+
+        /// <summary>
+        /// Fraction identifier for fractionated workflows. Returns 0 if not applicable.
+        /// </summary>
+        public int Fraction { get; }
+
+        /// <summary>
+        /// The plex or multiplex set identifier. Distinguishes between different isobaric
+        /// labeling experiments when multiple plexes are analyzed together (e.g., for bridge normalization).
+        /// </summary>
+        public int PlexId { get; }
+
+        /// <summary>
+        /// The isobaric channel label (e.g., "126", "127N", "127C", "128N" for TMT;
+        /// "113", "114", "115" for iTRAQ). Serves as the primary identifier for the sample
+        /// within a multiplexed experiment.
+        /// </summary>
+        public string ChannelLabel { get; }
+
+        /// <summary>
+        /// The reporter ion m/z value for this channel. Used to extract intensity values
+        /// from MS2/MS3 spectra during quantification.
+        /// </summary>
+        public double ReporterIonMz { get; }
+
+        /// <summary>
+        /// True if this channel is used as a reference or normalization channel
+        /// (e.g., pooled reference in TMT experiments).
+        /// </summary>
+        public bool IsReferenceChannel { get; }
+
+        /// <summary>
+        /// Creates a new isobaric quantification sample info.
+        /// </summary>
+        /// <param name="fullFilePathWithExtension">Full path to the source file.</param>
+        /// <param name="sampleIdentifier">Display name for the sample.</param>
+        /// <param name="condition">The condition or experimental group.</param>
+        /// <param name="biologicalReplicate">Biological replicate identifier.</param>
+        /// <param name="technicalReplicate">Technical replicate identifier.</param>
+        /// <param name="fraction">Fraction identifier.</param>
+        /// <param name="plexId">The plex identifier.</param>
+        /// <param name="channelLabel">The isobaric channel label (e.g., "126", "127N").</param>
+        /// <param name="reporterIonMz">The reporter ion m/z value for this channel.</param>
+        /// <param name="isReferenceChannel">True if this is a reference/normalization channel.</param>
+        public IsobaricQuantSampleInfo(
+            string fullFilePathWithExtension,
+            string sampleIdentifier,
+            string condition,
+            int biologicalReplicate,
+            int technicalReplicate,
+            int fraction,
+            int plexId,
+            string channelLabel,
+            double reporterIonMz,
+            bool isReferenceChannel)
+        {
+            FullFilePathWithExtension = fullFilePathWithExtension ?? string.Empty;
+            SampleIdentifier = sampleIdentifier ?? string.Empty;
```

Comment:
Sample identifier should likely always be populated. Having multiple quant sample infos that all have "" as their identifier doesn't feel good.